scripts/embargo_registrations.py

```diff
@@ -25,12 +24,12 @@
 
 
 def main(dry_run=True):
-    pending_embargoes = Embargo.find(Q('state', 'eq', Embargo.UNAPPROVED))
+    pending_embargoes = Embargo.objects.filter(state=Embargo.UNAPPROVED)
```

Comment:
I think `select_for_update` *is* appropriate here, since the embargoes are being updated.